master/reference/node/configuration.md

```diff
@@ -51,6 +51,22 @@ In addition to the above, `{{site.nodecontainer}}` also supports [the standard F
 >
 {: .alert .alert-info}
 
+### Node name determination
+
+The `{{site.nodecontainer}}` must know the name of the node on which it is running. The node name is used to
+retrieve the [Node resource](../calicoctl/resources/node) configured for this node if it exists, or to create a new node resource representing the node if it does not. It is
+also used to associate the node with per-node [BGP configuration](../calicoctl/resources/bgpconfig), [felix configuration](../calicoctl/resources/felixconfig), and endpoints.
+
+When launched, the `{{side.nodecontainer}}` container uses the following methods in order to determine
+what name to use:
```

Comment:
"uses the following methods in order to determine what name to use" --> "sets the node name according to the following order of precedence:"